OpenRA.Mods.Common/Scripting/Global/MediaGlobal.cs

```diff
@@ -189,21 +189,27 @@ public bool PlayMovieInRadar(string movie, LuaFunction playComplete = null)
 		}
 
 		[Desc("Display a text message to the player.")]
```

Comment:
We should mention what a default of `null` for `prefix` does.

Reply:
I'm open on suggestions about the wording.

Reply:
What about `"Display a text message to the player. If 'prefix' is nil the default mission prefix from the mod data is used."`?